src/compiler/scala/tools/nsc/typechecker/Namers.scala

```diff
@@ -1348,7 +1348,7 @@ trait Namers extends MethodSynthesis {
 
       // Add a () parameter section if this overrides some method with () parameters
       val vparamSymssOrEmptyParamsFromOverride =
-        if (overridden != NoSymbol && vparamSymss.isEmpty && overridden.alternatives.exists(_.info.isInstanceOf[MethodType])) ListOfNil // NOTEL must check `.info.isInstanceOf[MethodType]`, not `.isMethod`!
+        if (overridden != NoSymbol && vparamSymss.isEmpty && overridden.alternatives.exists(_.info.isInstanceOf[MethodType])) ListOfNil // NOTE: Must check `.info.isInstanceOf[MethodType]`, not `.isMethod`!
```

Comment:
`Must` should be `must` as it is a sentence fragment.  (If we're going to be really pedantic.)